docs/getting_started/bridge_console.md

```diff
@@ -3,59 +3,155 @@ title: Bridge
+
 :::info
-More details on Bridges and how it works can be found [here](https://docs.calimero.network/bridge/architecture).
+For more details on Bridges and their architecture, refer to the [Bridge Architecture ](https://docs.calimero.network/bridge/architecture) documentation.
 :::
 
 ## Permissions Management
 
-In order to use the Calimero bridge to transfer FTs, NFTs, or make cross-Shard calls, you'll need to set up bridge permissions.
-![](../../static/img/permissions.png)
+To ensure secure access and usage of the Calimero bridge for token transfers and cross-shard calls, it is important to manage permissions effectively. Permissions are used to control which accounts can utilize the bridge's features and perform specific actions.
+
+![](../../static/img/permissions-management.png)
+
+## FT and NFT connector regex rules
+
+The FT and NFT connectors allow you to define regex rules that determine which accounts are allowed to bridge tokens. This rule known as the allow regex rule, grant permission to specific accounts, enabling them to utilize the bridge for token transfers.
+
+To add an allow regex rule:
+
+- Click on the **Add new rule** button
+
+![](../../static/img/add-regex-ft-nft.png)
+
+- In the input field, enter a regex pattern that matches the desired accounts for bridging
+
+![](../../static/img/add-regex-rule.png)
+
+For example, if you want to allow testnet accounts, you can use the regex pattern `.*\.testnet`.  This regex rule allows any account ending with ".testnet" to bridge tokens using the FT or NFT connector.
+
+### Verify FT and NFT connector permissions
+
+To verify whether an account has the necessary permissions to use the FT or NFT connector:
+
+- Click on the **Check Permissions** button 
+
+![](../../static/img/check-permissions-nft-ft.png)
+
+- Provide the Account ID for which you want to verify the permissions
+
+![](../../static/img/check-permissions.png)
+
+## Cross-shard connector regex rules
+
+Cross-shard connectors have two rules: the account allow rule (similar to FT and NFT connectors) and the deny account rule.
+
+![](../../static/img/no-allow-xsc.png)
+
+### Cross-shard allow regex rule
+
+The account allow regex rule grants permission to specific accounts, enabling them to utilize the bridge for token transfers.
+
+To add an account allow regex rule:
+
+- Click on the **Add new rule** button
+- In the input field, enter a regex pattern that matches the desired accounts for bridging. 
+
+![](../../static/img/xsc-regex-allow.png)
+
+For example, if you want to allow testnet accounts, you can use the regex pattern `.*\.testnet`.
+
+This regex rule allows any account ending with ".testnet" to bridge tokens using the cross-shard connector.
+
+### Cross-shard deny regex rule
+
+The deny account rule explicitly denies access or permission to a specific account or set of accounts. It is used to restrict certain accounts from performing certain actions or operations.
```

Comment:
Would add something like:

The deny account rule explicitly denies access or permission to a specific account or set of accounts to a specific contract or set of contracts.